src/webhooks/models/webhooksFormatter.js

```diff
@@ -49,7 +50,14 @@ function slackWebhookFormat(message, options) {
 function teamsWebhookFormat(message) {
     return {
         themeColor: WEBHOOK_TEAMS_DEFAULT_THEME_COLOR,
-        text: message.replace(/\n/g, "   \n")
+        text: message.replace(/\n/g, '   \n')
+    };
+}
+
+function discordWebhookFormat(message){
+    return {
+        text: message,
+        username: WEBHOOK_SLACK_DEFAULT_REPORTER_NAME
```

Comment:
Looking at [DIscord webhook API # execute a webhook](https://discord.com/developers/docs/resources/webhook#execute-webhook) `text` should be changed to `content`

I have the code against a freshly created Discord server and I can confirm the webhook execution fails, once changed from `text` to `content`, it works.